### PR TITLE
Gradle: Upgrade to Kotlin 1.3.11

### PR DESCRIPTION
**Pull request by sschuberth**
See https://github.com/JetBrains/kotlin/releases/tag/v1.3.11.

Signed-off-by: Sebastian Schuberth <sebastian.schuberth@here.com>

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1102)

### CODE REVIEW

File: utils/src/main/kotlin/Hash.kt [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import ch.frankel.slf4k.*
+
+import com.here.ort.utils.hash
+import com.here.ort.utils.log
+import com.here.ort.utils.toHexString
+
+import java.io.File
+import java.util.Base64
+
+data class Hash(
```
Comment (fviernau) [inline utils/src/main/kotlin/Hash.kt]: Is this related?

Comment (sschuberth) [reply utils/src/main/kotlin/Hash.kt]: No, it's not, thanks for noticing.
